Initial install with yarn and Vuetify selected gives 6 problems

Hey! I wanted to say that Initial install with yarn and Vuetify selected gives 6 problems.
These are the options provided:
```
? Use a custom UI framework vuetify
? Choose rendering mode SPA
? Use axios module NO
? Use eslint YES
? Choose a package manager yarn
```

`VuetifyLogo.vue 6:3  error  Insert `/`  prettier/prettier`
```
index.vue   25:26  error    Insert `⏎············`                               prettier/prettier
  34:30  error    Insert `⏎············`                               prettier/prettier
  43:11  warning  Disallow self-closing on HTML void elements (<hr/>)  vue/html-self-closing
  47:11  warning  Disallow self-closing on HTML void elements (<br/>)  vue/html-self-closing
```
`inspire.vue  4:7  warning  Disallow self-closing on HTML void elements (<img/>)  vue/html-self-closing`

yarn lint --fix fixes it but then again gives error.

--- a/template/_.eslintrc.js
+++ b/template/_.eslintrc.js
@@ -13,12 +13,12 @@ module.exports = {
     parser: 'babel-eslint'
   },
   extends: [
+    '@nuxtjs',
     <%_ if (prettier) { _%>
     'prettier',
     'prettier/vue',
     'plugin:prettier/recommended',
     <%_ } _%>
-    '@nuxtjs',
     'plugin:nuxt/recommended'
   ],
   <%_ if (prettier) { _%>

